src/Core/wwwroot/css/reboot.css

```diff
@@ -458,3 +458,7 @@ progress {
 [hidden] {
     display: none !important;
 }
+
+fluent-button::part(control) {
```

Comment:
Why did you put this style in the reboot.css file? This file is not required (but optional)

Reply:
I put it in here to reflect the change in all templates. I can put it into `FluentButton.razor.css` if this fits better. The same needs to be done for the `FluentTextArea`.

Reply:
Just give me a moment. There I will fix the other components as well which I can find.

Reply:
@dvoituron should this also be applied to placeholders and entered text?
![grafik](https://github.com/user-attachments/assets/4afe52ab-a879-4ee7-b271-78f4a630cc9a)